calico.go

```diff
@@ -306,6 +307,32 @@ func cmdDel(args *skel.CmdArgs) error {
 		"Node":         nodename,
 	}).Info("Extracted identifiers")
 
+	calicoClient, err := CreateClient(conf)
+	if err != nil {
+		return err
+	}
+
+	ep := api.WorkloadEndpointMetadata{
+		Name:         args.IfName,
+		Node:         nodename,
+		Orchestrator: orchestrator,
+		Workload:     workload,
+	}
+
+	wep, err := calicoClient.WorkloadEndpoints().Get(ep)
+	if err != nil {
```

Comment:
This is a behaviour change. Before, if a endpoint couldn't be found, we'd still try to clean up the IP address. Now we won't.